packages/gatsby-plugin-image/package.json

```diff
@@ -0,0 +1,94 @@
+{
+  "name": "gatsby-plugin-image",
+  "version": "1.0.0",
+  "scripts": {
+    "build": "npm-run-all -s clean build:*",
+    "build:gatsby-node": "tsc --jsx react --downlevelIteration true --skipLibCheck true --esModuleInterop true --outDir dist/ src/gatsby-node.ts src/babel-plugin-parse-static-images.ts",
+    "build:gatsby-ssr": "microbundle -i src/gatsby-ssr.tsx -f cjs -o ./[name].js --no-pkg-main --jsx React.createElement --no-compress --external=common-tags,react --no-sourcemap",
+    "build:server": "microbundle -f cjs,es --jsx React.createElement --define SERVER=true",
+    "build:browser": "microbundle -i src/index.browser.ts -f cjs,modern,es --jsx React.createElement -o dist/gatsby-image.browser --define SERVER=false",
+    "build:server-compat": "microbundle -i src/components/compat.tsx -f cjs,es --jsx React.createElement -o dist/gatsby-image.compat --define SERVER=true",
+    "build:browser-compat": "microbundle -i src/components/compat.browser.tsx -f cjs,modern,es --jsx React.createElement -o dist/gatsby-image.compat.browser --define SERVER=false",
+    "watch": "run-p watch:*",
+    "watch:gatsby-node": "yarn build:gatsby-node --watch",
+    "watch:gatsby-ssr": "yarn build:gatsby-ssr watch",
```

Comment:
```suggestion
    "watch:gatsby-ssr": "yarn build:gatsby-ssr --watch",
```

Reply:
The `build:gatsby-ssr` script is using microbundle, which uses `watch`, not `tsc` which uses `--watch`